### ISSUE TITLE
[Feature Request] Support for Full Text Search

### ISSUE THREAD

**Issue by ohadeytan** [langchain-ai/langchain-milvus#30]
Hi, 

Milvus 2.5 introduced [`Full Text Search`](https://milvus.io/docs/full-text-search.md) which intend to be an enhanced BM25 interface and implementation.
Is adding this on `langchain-milvus` roadmap?

Thanks

**Comment by zc277584121** [comment]
@ohadeytan yes, I am now developing it. It' s a big refactor. maybe it will release within few weeks.

**Comment by ohadeytan** [comment]
@zc277584121 glad to hear. We will wait for it. Thanks!
Let me know if at some point you think we could help with that.

**Comment by zc277584121** [comment]
@ohadeytan Here is the code preview: https://github.com/langchain-ai/langchain-milvus/pull/33  
There are some tiny problems that need to be solved, and I am still working on it.
The how-to-use code will be like the `test_builtin_bm25_function()` in the unittest. Will write more detail usage documents later. 
Since it's a big refactor, we will be grad to get any pre-feedback if possible.

**Comment by ohadeytan** [comment]
Great! I will take a look.

**Comment by zc277584121** [comment]
close this issue since this feature merged : https://github.com/langchain-ai/langchain-milvus/pull/33
